app/processing/funding/index.js

```diff
@@ -0,0 +1,3 @@
+const getFundings = require('./get-fundings')
+
+module.exports = getFundings

```

Comment:
`getFundings` function for validate needed

Reply:
`getFundingsByCalculationId` does not need to return `calculationId` as it's not used anywhere. Remove here, schema and tests

Reply:
That is exactly what line 10 is doing by calling funding-schema.js which validates the whole array and elements in it as an atomic object.

https://github.com/DEFRA/ffc-pay-statement-constructor/blob/PAY-2318-aggregate-map-and-build-statement-constructor-object/app/processing/funding/fundings-schema.js